docs/pages/tutorials/embedded-mode-with-sqlite-nextjs.mdx

```diff
@@ -0,0 +1,277 @@
+import { Markdown } from '../../components/Page';
+
+# How to embed Keystone + SQLite in a Next.js app
+
+Keystone now supports SQLite! 🎉
```

Comment:
Using the term "now" pins this content to a point in time. Every day that passes the "now" becomes less relevant. Can we reframe this in a more timeless way? Something like:

```
Keystone uses Prisma to interact with your data, which means we're about to support SQLIte database
```